tool/tctl/common/bots_command.go

```diff
@@ -419,15 +419,15 @@ func (c *BotsCommand) updateBotLogins(bot *machineidv1pb.Bot, mask *fieldmaskpb.
 	desiredLoginsArray := utils.StringsSliceFromSet(desiredLogins)
 
 	if maps.Equal(currentLogins, desiredLogins) {
-		log.Infof("Logins will be left unchanged: %+v", desiredLoginsArray)
+		slog.InfoContext(ctx, "Logins will be left unchanged", "logins", desiredLoginsArray)
 		return nil
 	}
 
-	log.Infof("Desired logins for bot %q: %+v", c.botName, desiredLoginsArray)
+	slog.InfoContext(ctx, "Desired logins for bot", "bot", c.botName, "logins", desiredLoginsArray)
```

Comment:
Maybe out of scope for this PR, but I wonder if we should make a child logger that automatically includes the bot name in every log. It looks like we're currently only including it in a couple of logs and it seems like some common context all of these logs could include.